src/function.rs

```diff
@@ -59,6 +60,14 @@ impl Function {
   }
 }
 
+fn abs_path(context: &FunctionContext, path: &str) -> Result<String, String> {
+  let corrected_path = context.invocation_directory.join(path);
+  match corrected_path.canonicalize() {
```

Comment:
Call `.lexiclean()` instead of canonicalize. The paths that canonicalize produces are a bit weird.

Reply:
Changed!